controllers/hibp.js

```diff
@@ -50,7 +50,7 @@ async function notify (req, res) {
 
     const email = subscriber.email;
     // need to create unsubscribe button for these
-    const requestedLanguage = acceptedLanguages(subscriber.signup_language);
+    const requestedLanguage = subscriber.signup_language ? acceptedLanguages(subscriber.signup_language) : "";
```

Comment:
Why default to empty string instead of of "en" here?

Reply:
I thought I was cleverly avoiding hard-coding the default language, but it's hard-coded right below here anyway. Hmm ...